dis_snek/models/discord/guild.py

```diff
@@ -1035,13 +1035,13 @@ def get_thread(self, thread_id: Snowflake_Type) -> Optional["models.TYPE_THREAD_
             thread_id: The ID of the thread to get
 
         Returns:
-            Channel object if found, otherwise None
+            Thread object if found, otherwise None
+
+        Note:
+            get_channel can retrieve threads, so this is basically an alias with extra steps for that
 
         """
-        # get_channel can retrieve threads, so this is basically an alias with extra steps for that
-        if thread_id in self._thread_ids:
-            return self.get_channel(thread_id)
-        return None
+        return self.get_channel(thread_id)
```

Comment:
maybe we should show an error if the channel is valid, but not a thread type channel? Bcu now is a bit confusing that this method will basically allow any type of channel to be fetched.